docs/api/telegram.rst

```diff
@@ -1274,6 +1284,14 @@ about its business.
 
       .. versionadded:: 0.3
 
+   .. py:method:: delete()
+
+      Delete the message self.
```

Comment:
What? "Delete this message. The message can be deleted only if... It can also be deleted if it's one of the supported service messages."